.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -81,11 +82,11 @@ __REPLACE with OpenSearch wide initiatives to improve quality and consistency.__
 
 - [ ] Create [release tags](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/release-tag/release-tag.jenkinsfile) for each component (Jenkins job name: release-tag-creation).
 - [ ] Replace refs in [manifests/{{ env.VERSION }}](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}) with tags and remove checks.
+- [ ] If this is a major or minor version release, stop previous version's patch release.
```

Comment:
Maybe say we stop the previous patch version's building instead of release? WDYT

Reply:
Thanks reframed it a bit.